x-pack/plugins/secops/public/components/timeline/properties/index.tsx

```diff
@@ -170,7 +184,14 @@ export class Properties extends React.PureComponent<Props, State> {
           <EuiFlexItem grow={false}>
             <EuiPopover
               anchorPosition="downRight"
-              button={<EuiIcon type="gear" size="l" onClick={this.onButtonClick.bind(this)} />}
+              button={
+                <EuiIcon
+                  data-test-subj="settings-gear"
+                  type="gear"
+                  size="l"
+                  onClick={this.onButtonClick.bind(this)}
```

Comment:
bind is the same as `onClick={() => this.onButtonClick()}` right? I know we mix the two styles a bit ...
But the `() =>` is the most predominate. Just pointing it out, no changes asked.

Reply:
> bind is the same as `onClick={() => this.onButtonClick()}` right? I know we mix the two styles a bit ...

the general issue with the lambdas is they (in some cases) will cause React to unnecessary re-render a component

> Just pointing it out, no changes asked

I'll resolve this with no changes, and also ping you about the topic above

Reply:
👍 updated to remove the new references being created by the `bind(this)`